README.md

```diff
@@ -4,10 +4,10 @@ No code is the best way to write secure and reliable applications. Write nothing
 
 ## Getting Started
 
-Start by not writing any code.
+Start by writing any code.
 
 ```
-
+System.out.println("Hello, World!");
```

Comment:
The whole idea of NoCode is "NOCODE"!!!!

Reply:
He fucked us all